contrib/packaging/rpm/cfg/Dockerfile

```diff
@@ -1,24 +1,13 @@
-FROM fedora:24
+FROM fedora:25
 
-MAINTAINER "Andre Martins <andre@cilium.io>"
+LABEL "Maintainer: Marcin Skarbek <marcin@skarbek.name>"
 
-RUN dnf install -y gettext curl gcc fedora-packager fedora-review && \
-cd /tmp && \
-curl -Sslk -o go1.7.4.linux-amd64.tar.gz \
```

Comment:
1.8 will be in f26, f25 will only receive updates for 1.7. Why? Because I would like to propose cilium based on this spec to official repo.